deepmd/infer/model_devi.py

```diff
@@ -195,12 +195,13 @@ def make_model_devi(
     nframes_tot = 0
     devis = []
     for data in data_sets:
-        coord = data["coord"]
-        box = data["box"]
-        atype = data["type"][0] 
-        devi = calc_model_devi(coord, box, atype, dp_models, nopbc=nopbc)
-        nframes_tot += coord.shape[0]
-        devis.append(devi)
+        coords = data["coord"]
+        boxs = data["box"]
+        atype = data["type"][0]
```

Comment:
Is there a reason to only use the first (?) type?

Reply:
I think atom types for all frames in a dataset is the same, aren't they?

Reply:
I don't think we can publicly ensure it if the returned array is the size of `nframes*natoms` instead of `natoms`

Reply:
I agree with @njzjz , in this context we do not assume all frames of data["type"] has the same value.